app.rambox.ramboxce.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: app.rambox.ramboxce
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+add-extensions:
+  org.freedesktop.Platform.GL32:
```

Comment:
why do you need this?

Reply:
I changed it to GL.
Electron supports webgl and this package can be used to acess webgl content